app/views/common_parameters/welcome.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for(:title, _("Global Parameters")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("globe", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Global Parameters') %></h1>
+  <p><%= _("Foreman can pass two types of parameters to Puppet via the ENC (External Node Classifier) interface - global parameters (accessible from any manifest),
```

Comment:
This should not be so Puppet-specific, global parameters have a wide range of other uses, e.g. in templates. I'd suggest removing all references to Puppet.

It would be better to indicate that adding global parameters here at the top level may allow them to be overridden elsewhere.

The references to class parameters are also irrelevant here.